docs/advanced/UsageWithReactRouter.md

```diff
@@ -1,5 +1,7 @@
 # Usage with React Router
 
+Warning: This guide assumes you are using `react-router@^2.7.0`. As of May 2017 the current version of React Router is 4+. If you have just run `npm install --save react-router`, you will be using a version that does not match this guide.
```

Comment:
This applies to both 2.x and 3.x. They are the same, just with different deprecations.